go/nbs/table_set.go

```diff
@@ -228,20 +276,19 @@ func (ts tableSet) Rebase(specs []tableSpec) tableSet {
 	}
 
 	// Open all the new upstream tables concurrently
-	openedTables := make(chunkSources, len(tablesToOpen))
+	merged.upstream = make(chunkSources, len(tablesToOpen))
```

Comment:
you're right! I dropped the code that did that when I changed approaches this morning. Fixed